src/sst/core/componentInfo.cc

```diff
@@ -143,12 +143,14 @@ ComponentInfo::ComponentInfo(ComponentInfo &&o) :
 ComponentInfo::~ComponentInfo() {
     if ( link_map ) delete link_map;
     if ( component ) {
+#ifndef SST_ENABLE_PREVIEW_BUILD        
         // For backward compatibility, don't delete component defined
```

Comment:
Similar to the other case, should the 'if' statement be ifdefed but not the contents? So all component defined subcomponents get deleted?

Reply:
Fixed and pushed.